config/tsconfig.base.json

```diff
@@ -14,6 +14,7 @@
     "noUnusedLocals": true,
     "sourceMap": true,
     "stripInternal": true,
-    "target": "ES2015"
+    "target": "ES2015",
+    "esModuleInterop": true
```

Comment:
`protobufjs@7.2.4` requires `long@5.0.0` which is a esm now, we need to enable this rule to import it properly.
https://github.com/dcodeIO/long.js/commit/86410397c36ad64aa6c9a45293bff918ce0ea14d